jobscripts/peregrine/installjob_skylake.sh

```diff
@@ -4,8 +4,8 @@
 #SBATCH --cpus-per-task=2
 #SBATCH --mem=64G
```

Comment:
Should we also just request all the cores+mem?

Reply:
Yes, there are 12 cores on the V100s, right?